src/System.Private.ServiceModel/tools/test/Bridge/WcfTestBridgeCommon/CertificateGenerator.cs

```diff
@@ -216,22 +216,28 @@ public TimeSpan ValidityPeriod
             }
         }
 
-        public X509CertificateContainer CreateMachineCertificate(params string[] subjects)
+        public X509CertificateContainer CreateMachineCertificate(CertificateCreationSettings creationSettings = null)
```

Comment:
NIT: It seems really odd from an API standpoint that we could call `CreateMachineCertificate` without providing any params. We probably should mandate that it's not null.